### PR TITLE
feat: Add resource_reports field in Table API  ( Atlas proxy)

### PR DESCRIPTION
**Pull request by dechoma**
### Summary of Changes

Add resource_reports field in Table API ( atlas proxy)  to support integration with tools generating static HTML reports, such as [popmon](https://github.com/ing-bank/popmon), [pandas-profiling]( https://github.com/pandas-profiling/pandas-profiling) or [great_expectations](https://github.com/great-expectations/great_expectations)..

### Tests

Updated existing unit tests

### Documentation

N/A

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`

Resolve lyft/amundsen#525

### CODE REVIEW

File: metadata_service/proxy/atlas_proxy.py [hunk 1]
```diff
@@ -375,6 +377,23 @@ def get_table(self, *, table_uri: str) -> Table:
                     )
                 )
 
+            reports = []
+            for report in attrs.get("reports") or list():
+                LOGGER.info("Getting matadata for table Report GUID:{}".format(report.get("guid")))
+                report_entity = self._driver.entity_guid(report.get("guid"))
```
Comment (mgorsk1) [inline metadata_service/proxy/atlas_proxy.py:383]: you could get all the reports with one request instead of doing per-report-request

Comment (dechoma) [reply metadata_service/proxy/atlas_proxy.py:383]: You're right, I'll use entity_bulk

**Comment by jornh** [comment]
@dechoma looks like a few adjustments are needed to keep Prettier happy, like: https://travis-ci.com/github/lyft/amundsenfrontendlibrary/jobs/353457956#L742-L744

**Comment by mgorsk1** [comment]
@verdan can you check if that's aligned with our latest conversations ?

**Comment by verdan** [comment]
@dechoma @mgorsk1 I believe this PR can be closed. As we discussed, we are not going to implement it like this, and have this get table reports method configurable, so that community can have their own implementations.

**Comment by dechoma** [comment]
Closing

**Pull request closed without merging**